src/Units/APCModel.java

```diff
@@ -16,7 +16,7 @@
 
   public APCModel()
   {
-    super("APC", UnitEnum.APC, 5000, 70, 0, 6, moveType, actions, healHabs, null);
+    super("APC", UnitEnum.APC, ChassisEnum.TRUCK, 5000, 70, 0, 6, moveType, actions, healHabs, null);
```

Comment:
I'm curious why you have both tank and truck. I'm also curious why APC is a truck, since it is a treaded vehicle. I would sort of see a more natural distinction as being in terms of domain - land, sea, air (maybe low-air and high-air to distinguish copters and jets), subsea. Do we gain anything by distinguishing truck and tank (and how do we determine which is which)?

Reply:
It's explained in UnitModel. I called anything that takes more than 9% from an infantry a truck. That distinguishes it from tire/tread, since chassis distinctions are likely going to be used in damage, and those vehicles are a lot less armored.